packages/fx-core/src/component/utils/metadataGraphPermssion.ts

```diff
@@ -0,0 +1,89 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import path from "path";
+import fs from "fs-extra";
+import isUUID from "validator/lib/isUUID";
+import { MetadataV3 } from "../../common/versionMetadata";
+import { ProjectModel } from "../configManager/interface";
+import { AADManifest } from "../driver/aad/interface/AADManifest";
+import { getDetailedGraphPermissionMap, graphAppId, graphAppName } from "../driver/aad/permissions";
+import { actionName } from "../driver/teamsApp/configure";
```

Comment:
should it be aadApp/update's action name?

Reply:
Fixed it. Thanks.